parm/wflow/plot.yaml

```diff
@@ -12,37 +12,45 @@ default_task_plot: &default_task
     PDY: !cycstr "@Y@m@d"
     cyc: !cycstr "@H"
     subcyc: !cycstr "@M"
+    fhr: '#fhr#'
     LOGDIR: !cycstr "&LOGDIR;"
     SLASH_ENSMEM_SUBDIR: '&SLASH_ENSMEM_SUBDIR;'
     ENSMEM_INDX: '#mem#'
-    nprocs: '{{ nnodes * ppn }}'
+    nprocs: '{{ parent.nnodes * parent.ppn }}'
+  join: !cycstr '&LOGDIR;/{{ jobname }}_@Y@m@d@H&LOGEXT;'
   native: '{{ platform.SCHED_NATIVE_CMD }}'
-  nnodes: 1
   nodes: '{{ nnodes }}:ppn={{ ppn }}'
-  partition: '{% if platform.get("PARTITION_DEFAULT") %}&PARTITION_DEFAULT;{% else %}None{% endif %}'
+  nnodes: 2
   ppn: 24
+  partition: '{% if platform.get("PARTITION_DEFAULT") %}&PARTITION_DEFAULT;{% else %}None{% endif %}'
```

Comment:
Please put these back in alphabetical order.

Reply:
done, updated.